bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/TpeEditorTools.java

```diff
@@ -136,25 +122,32 @@ private boolean isEnabled() {
         return ctx.progShell().isDefined() && ctx.obsShell().isDefined() && OTOptions.isProgramEditable(ctx.progShell().get()) && OTOptions.isObservationEditable(ctx.obsShell().get());
     }
 
+    private boolean isAuto() {
+        return ImOption.fromScalaOpt(_tpe.getImageWidget().getContext().targets().env())
+                .exists(te -> te.getGuideEnvironment().getPrimary().isAutomatic());
+    }
+
     /**
-     * Update the enable states of the buttons based on the OT editable state.
+     * Update the enable states of the buttons based on the OT editable state and whether or not the group is
+     * the automatic guide group.
      */
     public void updateEnabledStates() {
         final boolean enabled = isEnabled();
+        final boolean notAuto = !isAuto();
         _dragButton.setEnabled(enabled);
-        _eraseButton.setEnabled(enabled);
+        _eraseButton.setEnabled(enabled && notAuto);
```

Comment:
A minor niggling thing but it's sometimes easier to follow if we can avoid the logic flip.  That is

````java
_eraseButton.setEnabled(enabled && manual);
````